packages/amplify-e2e-tests/src/__tests__/analytics.test.ts

```diff
@@ -1,4 +1,4 @@
-import { initJSProjectWithProfile } from '../init';
+import { initJSProjectWithProfile, amplifyPushUpdate, deleteProject } from '../init';
```

Comment:
Is this a part of this PR?

Reply:
Yes, it is to clean up resources

Reply:
That should be a part of another PR then..doesn't look like an analytics fix

Reply:
It is a part of the test for Analytics

Reply:
Nah, deleted my comment.